bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -167,7 +167,7 @@ spec:
               value: {{ (include "postgresql-ha.repmgrDatabase" .) | quote }}
             {{- end }}
             - name: POSTGRESQL_NUM_SYNCHRONOUS_REPLICAS
-              value: {{ .Values.postgresql.replicaCount | quote }}
+              value: {{ sub .Values.postgresql.replicaCount 1 | quote }}
```

Force integer conversion:
```suggestion
              value: {{ sub (int .Values.postgresql.replicaCount) 1 | quote }}
```